Classes/Mvc/Request.php

```diff
@@ -33,12 +33,12 @@
 class Request
 {
     /**
-     * @var array
+     * @var mixed[]
```

Comment:
mixed[] vs. array communicates to the reader that the programmer has thought about the type and that allows for more than one type in the array. ("array" communicates "I haven't thought about the types in the array".)